TODO.md

```diff
@@ -16,3 +16,10 @@
 ** read log
 ** show static daily price graph
 ** show dynamic daily price graph
+
+
+### FINISHED
+* make `suggestion` a dict and add reason for suggestion
+* order quantity be half (or some percentage) of current shares of holding
+* log suggestion
+* log the change of daily extremes (and the reason)
```

Comment:
Can you try using [task list](https://help.github.com/articles/about-task-lists/#creating-task-lists)?